src/hotspot/share/gc/shenandoah/shenandoahBarrierSet.inline.hpp

```diff
@@ -100,31 +100,35 @@ inline oop ShenandoahBarrierSet::load_reference_barrier(oop obj) {
 
 template <DecoratorSet decorators, class T>
 inline oop ShenandoahBarrierSet::load_reference_barrier(oop obj, T* load_addr) {
+  if (obj == nullptr) {
+    return nullptr;
+  }
```

Comment:
Let's use `NULL` consistently? Yes, it is an old style, but it is consistent with the code in the same file.